apps/src/code-studio/pd/foorm/editor/components/FoormEntityLoadButtons.jsx

```diff
@@ -28,14 +31,45 @@ class FoormEntityLoadButtons extends React.Component {
     setLastSavedQuestions: PropTypes.func
   };
 
+  state = {
+    latestVersionsOnly: true
+  };
+
+  shouldShowLatestVersionsOnly() {
+    return this.props.showVersionFilterToggle && this.state.latestVersionsOnly;
+  }
+
+  /**
+   * Takes props.foormEntities in the form of {metatadata: {name: string, version: number}, text: string}
+   * and optionally sorts and filters (based on version filter toggle) and renders them as MenuItems.
+   * @returns Array<MenuItem>
+   */
   getDropdownOptions() {
-    return this.props.foormEntities.map((entity, i) => {
-      return this.renderMenuItem(
-        () => this.props.onSelect(entity['metadata']),
-        entity['text'],
-        i
-      );
-    });
+    const latestVersionMap = getLatestVersionMap(this.props.foormEntities);
+
+    return (
+      this.props.foormEntities
```

Comment:
Does this sort work for library questions as well?

Reply:
the version evaluates to NaN, but they should have unique names, so it works out. if the names were the same, the sort would still run, it'd just be an undefined order